### ISSUE TITLE
Provide a 3D data sample in GeoServer and publish it in a MS map (lower priority)

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/MapStore2-c126#34]

**Comment by tdipisa** [comment]
Waiting the client for the sample of DEM to publish in GS/MS

**Comment by tdipisa** [comment]
Blocked by the client
